ophyd/commands.py

```diff
@@ -236,6 +236,16 @@ def mov(positioner, position):
                 flag += 1
 
     print(tc.Normal + '\n')
+    for err in [s for s in stat if s.success == False]:
```

Comment:
minor point: `if not s.success` is preferred so that anything "false-y" would work correctly here.